src/platforms/android/configuration/integrations/okhttp.mdx

```diff
@@ -110,3 +110,61 @@ Learn more about manually capturing an error or message, in our <PlatformLink to
 </Note>
 
 To view and resolve the recorded message, log into [sentry.io](https://sentry.io) and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
+
+## Customize the Recorded Span
```

Comment:
let's add a note that this is available thru the beta version, similar to https://github.com/getsentry/sentry-docs/blame/a8ce23144605efb58f44e2ccbafed4f41d2d8aef/src/platforms/java/guides/spring/advanced-usage.mdx#L36-L40